zebra-state/src/lib.rs

```diff
@@ -69,6 +69,13 @@ pub struct Config {
 
     /// The maximum number of bytes to use caching data in memory.
     pub memory_cache_bytes: u64,
+
+    /// Whether to use an ephemeral, in-memory store instead of writing to disk.
+    ///
+    /// Set to `false` by default. If this is set to `true`, [`cache_dir`] is ignored and no state
+    /// is written to disk.  This is useful for testing but is otherwise probably not what you
+    /// want, unless you intend to load the entire blockchain into memory.
```

Comment:
I would personally remove the last line because it can be useful in general for stateless/volatile usage.

```suggestion
    /// is written to disk (meaning the entire blockchain may be loaded into memory).
```